Source/Build Specs/MeasurementLink Service.vipb

```diff
@@ -1,7 +1,7 @@
-<VI_Package_Builder_Settings Version="2020.1" Created_Date="2023-01-23 14:05:21" Modified_Date="2023-11-14 15:14:33" Creator="lvadmin" Comments="" ID="d6a8d70e3546d75f7536825b8c1e1299">
+<VI_Package_Builder_Settings Version="2020.1" Created_Date="2023-01-23 14:05:21" Modified_Date="2023-11-16 10:10:39" Creator="lvadmin" Comments="" ID="8e7965136f751ce2ff2ce750b3189051">
   <Library_General_Settings>
     <Package_File_Name>NI_MeasurementLink_Service</Package_File_Name>
-    <Library_Version>1.3.0.4</Library_Version>
+    <Library_Version>1.3.0.2</Library_Version>
```

Comment:
This version has been incremented unintentionally for various reasons. I put it back to the version it should be at.